changelogs/current.yaml

```diff
@@ -391,6 +391,9 @@ new_features:
 - area: geoip
   change: |
     Added support for :ref:`Maxmind geolocation provider <envoy_v3_api_msg_extensions.geoip_providers.maxmind.v3.MaxMindConfig>`.
+- area: tracing
+  change: |
+    Datadog propagation styles now includes ``B3Multi``, which replaces ``B3``. ``B3`` is marked as deprecated and should no longer be used.
```

Comment:
Please refine the comment to clarify that this is part of an upgrade of the datadog tracing library.